app/src/main/java/com/example/robin/roomwordsample/Adapter/WordListAdapter.kt

```diff
@@ -6,10 +6,8 @@ import android.graphics.Paint
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import android.widget.Button
-import android.widget.CheckBox
-import android.widget.RelativeLayout
-import android.widget.TextView
+import android.view.inputmethod.InputMethodManager
+import android.widget.*
```

Comment:
Avoid wildcard imports

Reply:
@rob729 I tried to 😅️ But Android Studio automatically converts to wildcard, may be because many classes under that package are imported.

Reply:
Ok. If that is the case then no need to change it.